### PR TITLE
[FEATURE] Allow loading mappings from paths

### PR DESCRIPTION
**Pull request by FoxxMD**
Fluent will check the "paths" node in the doctrine config for an array of directories which it should load mapping files from
### Changes proposed in this pull request:
- Fluent will check the "paths" node in the doctrine config for an array of directories which it should load mapping files from and then load them

### CODE REVIEW

File: src/FluentDriver.php [hunk 1]
```diff
@@ -117,6 +127,103 @@ public function getMappers()
...
+        $contents = file_get_contents($pathToFile);
+
+        //Start with a blank namespace and class
+        $namespace = $class = '';
+
+        //Set helper values to know that we have found the namespace/class token and need to collect the string values after them
+        $getting_namespace = $getting_class = false;
+
+        //Go through each token and evaluate it as necessary
+        foreach (token_get_all($contents) as $token) {
```
Comment (guiwoda) [inline src/FluentDriver.php]: @FoxxMD this is why I didn't add the file parser right away. Tokenizing a file's contents seems overkill, when the alternative is to list the FQCNs in the config and be done with it.
I'm not comfortable with the current solution, I don't know what @patrickbrouwers thinks about it.

Comment (FoxxMD) [reply src/FluentDriver.php]: The alternative is adding 80+ lines to my doctrine config, for each manager. It gets unwieldy.  Plus with paths I don't have to update my doctrine config each time I add a new mapping, just add the mapping and be done with it.

Comment (FoxxMD) [reply src/FluentDriver.php]: Let me see if I can get doctrine's way to work...

File: src/FluentDriver.php [hunk 2]
```diff
@@ -117,6 +127,45 @@ public function getMappers()
...
+
+            $iterator = new \RecursiveIteratorIterator(new \RecursiveDirectoryIterator($path), \RecursiveIteratorIterator::LEAVES_ONLY);
+
+            foreach ($iterator as $file) {
+                if ($file->getBasename('.php') == $file->getBasename()) {
+                    continue;
+                }
+
+                $sourceFile = realpath($file->getPathName());
+                require_once $sourceFile;
```
Comment (FoxxMD) [inline src/FluentDriver.php:152]: @guiwoda  This is how [doctrine](http://www.doctrine-project.org/api/common/2.4/source-class-Doctrine.Common.Persistence.Mapping.Driver.StaticPHPDriver.html#82-133) finds classes for the StaticPHPDriver

**Comment by patrickbrouwers** [comment]
Keeping this open as reminder, but I'd prefer a different solution.

**Comment by patrickbrouwers** [comment]
Replaced by #48

**Pull request closed without merging**